bqplot/figure.py

```diff
@@ -125,7 +125,7 @@ class Figure(DOMWidget):
     layout = Instance(Layout, kw={
             'flex': '1',
             'align_self': 'stretch',
-            'max_width': '700px'
```

Comment:
I feel setting `min_width` rather than `max_width` by default as, globally, we are trying to expand whenever there is space available. Setting `max_width` prevents that.